service-manual/measurement/completion-rate.md

```diff
@@ -20,117 +20,74 @@ breadcrumbs:
     url: /service-manual/measurement
 ---
 
-When users are unable to complete a digital transaction, they can be pushed to use other channels. This leads to low levels of digital take-up and customer satisfaction, and a higher cost per transaction.
+Completion rate shows how many people manage to use the service successfully.
 
-Measuring end-to-end completion rates helps to identify whether users have problems completing a transaction. Analysing the drop-out rate at each step of a transaction will pinpoint the specific processes that users fail to complete.
-
-## What you will be measuring
-
-The completion rate measures the proportion of people who start a transaction and are able to complete it.
-
-A transaction is a self-contained process that's defined by the service manager, and is typically only completed when a user has been through the whole thing end to end.
-
-A transaction completed only partially online means the user may complete a single process within the overall transaction, like booking an appointment or completing a part of an application.
+Once your service goes live you’ll find steps that cause users to give up, and bugs that make the service impossible to complete. And as you fix these issues, your completion rate will rise.
 
 ## How to measure completion rate
 
-1.    Work out the number of completed transactions.
-2.    Divide it by the number of started transactions, expressed as a percentage.
+1. Count the number of completed transactions
+2. Divide it by the number of started transactions, as a percentage
 
-### Before the transaction starts
+## What is a completed transaction?
 
-Before the transaction has started, users should clearly be told:
+A transaction is completed when **the service did what it was supposed to do**
```

Comment:
Ambiguous. People:

- finding out early in the journey that the service they're using only applies to a particular target population
- abandoning the service

is what the service is supposed to do. Is that completion?

What about:

    A transaction is completed when the user has achieved their desired outcome, eg booked a driving test.